cluster/examples/workloads/kubernetes/wordpress/aws/gitops/aws-infra/eks-class.yaml

```diff
@@ -3,8 +3,11 @@ apiVersion: compute.aws.crossplane.io/v1alpha2
 kind: EKSClusterClass
 metadata:
   name: standard-eks-cluster
+  labels:
+    example: "true"
 specTemplate:
-  region: us-west-2
+  writeConnectionSecretsToNamespace: crossplane-system
+  region: us-east-1
```

Comment:
This region change was just for my convenience, not required for simple resource class selection.